ticdc: Include local:// and s3:// sink support (#5520)
pingcap/docs: ticdc/manage-ticdc.md

For more parameters of Pulsar, see [pulsar-client-go ClientOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ClientOptions) and [pulsar-client-go ProducerOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ProducerOptions).

#### Configure sink URI with cdclog

> The cdclog files can be used together with `br` to provide point-in-time (PiTR) recovery. This is experimental feature.

Reviewer: @amyangfei Here "cdclog" is mentioned for the first time in the entire docs repo. I think we need to explain what "cdclog" is, what it is used for, etc.

> The cdclog files can be used together with `br` to provide point-in-time (PiTR) recovery. This is experimental feature.

Reviewer: Is the "cdclog file" here referring to "Files written on the local filesystem or on Amazon S3 compatible storage"?

Author: `cdclog` are files written by TiCDC. CDC stands for change data capture.

Reviewer: ```suggestion
The `cdclog` files (files written by TiCDC) can be used together with Backup & Restore (BR) to provide point-in-time (PiTR) recovery. This is an experimental feature.
```

> The cdclog files can be use together with `br` to provide point-in-time (PITR) recovery. This is experimental functionality.

Reviewer 2: ```suggestion
The cdclog files can be used together with `br` to provide point-in-time (PiTR) recovery. This is experimental feature.
```

> The `cdclog` files (files written by TiCDC) can be used together with Backup & Restore (BR) to provide point-in-time (PiTR) recovery. This is an experimental feature.

Reviewer: ```suggestion
The `cdclog` files (files written by TiCDC on the local filesystem or on the Amazon S3 compatible storage) can be used together with Backup & Restore (BR) to provide point-in-time (PITR) recovery. See [Point in Time recovery (experimental feature)](/br/use-br-command-line-tool.md#point-in-time-recovery-experimental-feature) for details.
```

> See `br restore cdclog --help` for more information on this command.

Reviewer: I think this sentence would be more useful if it is put under "Point in Time recovery (experimental feature)".

{{< copyable "shell-regular" >}}

```shell
cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="local:///data/cdclog" --config changefeed.toml
```

{{< copyable "shell-regular" >}}

```shell
cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="s3://logbucket/test?endpoint=http://$S3_ENDPOINT/" --config changefeed.toml

> ```

Reviewer: I think we should describe what the two commands are about. ("local:///data/cdclog" and "s3://logbucket/test?endpoint=http://$S3_ENDPOINT/")

For more parameters of Pulsar, see [pulsar-client-go ClientOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ClientOptions) and [pulsar-client-go ProducerOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ProducerOptions).

#### Configure sink URI with cdclog

> The `cdclog` files (files written by TiCDC on the local filesystem or on the Amazon S3 compatible storage) can be used together with Backup & Restore (BR) to provide point-in-time (PITR) recovery. See [Point in Time recovery (experimental feature)](/br/use-br-command-line-tool.md#point-in-time-recovery-experimental-feature) for details.

Reviewer: ```suggestion
The `cdclog` files (files written by TiCDC on the local filesystem or on the Amazon S3-compatible storage) can be used together with Backup & Restore (BR) to provide point-in-time (PITR) recovery. See [Point in Time recovery (experimental feature)](/br/use-br-command-line-tool.md#point-in-time-recovery-experimental-feature) for details.
```

> {{< copyable "shell-regular" >}}
> 
> ```shell
> cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="local:///data/cdclog" --config changefeed.toml
> ```
> 
> This command creates a changefeed that would write cdclog files localy to `/data/cdc/log`.

Reviewer: The GitHub comment does not show the ending ``` ``` ```  in the suggestion mode.
```suggestion
The following command creates a changefeed that will write cdclog files locally to the `/data/cdc/log` directory.

{{< copyable "shell-regular" >}}

```shell
cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="local:///data/cdclog" --config changefeed.toml
```
```

> {{< copyable "shell-regular" >}}
> 
> ```shell
> cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="s3://logbucket/test?endpoint=http://$S3_ENDPOINT/" --config changefeed.toml
> ```
> 
> This command creates a changefeed that would write cdclog files to external S3 storage in the `logbucket` bucket with a subdirectory of test. The endpoint is set in the URI, this is needed if you are using S3 compatible storage instead of Amazon S3.

Reviewer: ```suggestion
This command creates a changefeed that will write cdclog files to an external S3 storage in the `logbucket` bucket with a subdirectory of `test`. The endpoint is set in the URI, which is needed if you are using an S3-compatible storage other than Amazon S3.

{{< copyable "shell-regular" >}}

```shell
cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="s3://logbucket/test?endpoint=http://$S3_ENDPOINT/" --config changefeed.toml
```
```
